lib/hooks/useContributorPullRequestsChart.ts

```diff
@@ -42,6 +42,10 @@ const useContributorPullRequestsChart = (contributor: string, topic: string, rep
         },
       },
     ],
+    tooltip: {
+      trigger: "axis",
+      formatter: "<strong>{b}</strong> days ago <br/> <strong>{c}</strong> PRs",
```

Comment:
Realized. This will not work for "1 days ago."

We will need a check for singular 

b == 1 ? "day ago" : "days ago"

Reply:
Got it! Thanks for pointing that out. Added this check and pushed the changes.